wasm/index.html

```diff
@@ -8,22 +8,22 @@
         text-align: center;
       }
       #gotherworld {
-        /* TODO width/height might be ok if set in go */
-        width: 320px;
-        height:200px;
+        width: 640px;
+        height: 400px;
         display: block;
         margin-left:auto;
         margin-right:auto;
         -webkit-box-shadow: 10px 10px 5px 0px rgba(0,0,0,0.75);
         -moz-box-shadow: 10px 10px 5px 0px rgba(0,0,0,0.75);
         box-shadow: 10px 10px 5px 0px rgba(0,0,0,0.75);
+        image-rendering: pixelated;
       }
     </style>
   </head>
   <body>
     <!-- TODO use fancy webworkers for caching all the assets... -->
     <h1><img src="logo.png" alt="Gother World"></h1>
-    <canvas id="gotherworld"/>
+    <canvas id="gotherworld" width="320" height="200"/>
```

Comment:
minor: something like `canvasElement.Set("width", 320)` and ... in js-canvas.go might also work if we want to leverage consts in go.

Reply:
yes we could but the game itself is fixed at this resolution